code/__defines/items_clothing.dm

```diff
@@ -77,7 +77,8 @@
 #define BLOCK_GAS_SMOKE_EFFECT     0x10 // Blocks the effect that chemical clouds would have on a mob -- glasses, mask and helmets ONLY! (NOTE: flag shared with ONESIZEFITSALL)
 #define FLEXIBLEMATERIAL           0x20 // At the moment, masks with this flag will not prevent eating even if they are covering your face.
 #define COVER_PREVENT_MANIPULATION 0x40 // Only clothing with this flag will prevent manipulation under it. Its for space suits and such, unlike from usual Bay12 rules of clothing manipulation.
-
+#define DRAG_N_DROP_UNEQUIP        0x80 // Allow you put intems in hands with drag and drop
+#define LOUDLY_EQUIPEMENT          0x100// Play sound when equipped/unequipped
```

Comment:
This should be something like HAS_EQUIP_SOUND
Although is a flag even necessary? Couldn't we maybe just check whether an equipsound var is populated or not?

Reply:
We actually can. 
But it can be only TRUE or FALSE, what's the reason to make it variable instad flag?

Reply:
@LudwigVonChesterfield 
> What if we have different equip/unequip sounds?

We will add that var then.